dace/codegen/targets/cuda.py

```diff
@@ -152,8 +152,6 @@ def get_generated_codeobjects(self):
 #include <cuda_runtime.h>
 #include <dace/dace.h>
 
-{file_header}
-
```

Comment:
Why?

Reply:
It doesn't work even for the tests with this line...